### ISSUE TITLE
[1.2.0] Cannot read from measurement if its name is starting with an underscore.

### ISSUE THREAD

**Issue by mariusoe** [influxdata/influxdb#8044]
### Bug report

Hey,

in  InfluxDB version 1.2.0 it is not possible to read from measurements starting with an underscore. If the name of a measurement starts with "_" the response is always empty. In a previous version (e.g. 1.0.1), the measurements could be queried as expected.

__System info:__ Windows 7 / InfluxDB 1.2.0

__Steps to reproduce:__

1. Create a measurement starting with an underscore. E.g.: _data
2. Reading from the measurement. E.g.: SELECT * FROM _data

__Expected behavior:__ The data contained in the measurement

__Actual behavior:__ An empty response

**Comment by e-dard** [comment]
Thanks for the report @mariusoe.

Confirmed:

```
 // ⇒  influxd-v1.1.0

> insert _hello value=1
> select * from _hello
name: _hello
time                value
----                -----
1488198319116228104 1
```

```
// ⇒  influxd-v1.2.1-rc5

> insert _hello value=1
> select * from "_hello"
> select * from _hello
> select * from "db".."_hello"
> show measurements
name: measurements
name
----
_hello
cpu

>
```

**Comment by e-dard** [comment]
Possible dupe of #7961.

**Comment by gerrickw** [comment]
Have the same issue, is there a way to workaround this issue?  I have a bunch of measurements that start with '_'.

**Comment by e-dard** [comment]
Fixed via #8069
